docs/fast_data/faq/benchmark_fast_data.md

```diff
@@ -0,0 +1,155 @@
+---
+id: benchmark_fast_data
+title: Bootstrapping Fast Data
+sidebar_label: Bootstrapping Fast Data
+---
+
+This section provides guidelines and answers on how to provision the infrastructure components in order to have an up-and-running instance of Mia-Platform Fast Data. 
+
+In particular, it lists some of the minimum resources in terms of CPU or RAM that the components (Kafka, MongoDB and Mia-Platform Plugins) needs to satisfy, along with their configuration.
+
+:::info Before you start
+
+Fast Data solution can either be installed in a [PaaS](/paas/overview.md) or [Self-Hosted](/self_hosted/self_hosted_requirements.md) setup. The goal of this guide is to provide best practices that are independent of the infrastructure.
+
+The level of details of the following answers assumes that you have basic knowledge about the aforementioned components. 
+:::
+
+## Assumptions
+
+This table shows details for each component that has been monitored and analyze to provide the guidelines.
+
+<table>
+    <tr><th>Component</th><th>Replicas</th><th>Details</th></tr>
+    <tr><td>Kafka</td><td>4 brokers</td><td> 600.000 nominal events X day, i.e. 30GB of ingestion messages.</td></tr>
+    <tr><td>MongoDB</td><td>3 replica sets</td><td>Enterprise distribution having 32 GB of RAM.</td></tr>
+    <tr><td>Fast Data</td><td>6 nodes</td><td>K8S cluster with each node having 8 core.</td></tr>
+</table>
+
+
+## Kafka
+
+
+### What is the topic setup I should use to guarantee an high rate of ingestion?
+
+
+When creating and managing topics in a Kafka cluster, is crucial to define a number of partitions that can balance the identifiers of messages. This means that a topic:
+
+* linked to a data source having a fixed cardinality, requires a number of partitions `<=` 3;   
+* linked to a data source having an higher number of insert/update/delete operation requires a number of partition `>` 3.
+
+:::tip Use Case
+A topic having an ingestion rate of `2.2k message/minute` requires 10 partitions to distribute the messages among the topics while speeding up micro-services performances.
```

Comment:
"How did we get to this? A single microservice handles 220 messages per minute? If so, let's start by discussing the capacity of the individual service and then provide some examples. I would also add some details about the size of the messages (if relevant).

Reply:
hi @akelity yep i agree with you, the topic partitioning is just a rule of thumb. i made the section referencing a blog post by confluent which goes into further details about sizing of messages and stuff like this